interfaces/builtin/greengrass_support.go

```diff
@@ -169,12 +332,16 @@ const greengrassSupportConnectedPlugSeccomp = `
 
 # allow use of ggc_user and ggc_group
 # FIXME: seccomp arg filter by this uid/gid when supported by snap-confine
+lchown
 fchown
 fchown32
 fchownat
 setgroups
 setgroups32
 
+# for joining another namespace
+setns
```

Comment:
Which ones? From network-control you get:
```
setns - CLONE_NEWNET
```

Please enumerate the namespaces greengrass is using (you can see these via strace) or if it is using all of them, mention that in the comment.

Reply:
The only namespace I see with strace is just `CLONE_NEWNS`. I'll update the rule here to just add that, but how does the seccomp compiler handle multiple syscall lines with different arguments? Do they get combined in the rule that runs or do they conflict with each other?